jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/BuildStepsIntegrationTest.java

```diff
@@ -83,7 +83,7 @@ public void testSteps_forBuildToDockerRegistry()
         new Command("docker", "inspect", imageReference).run(),
         CoreMatchers.containsString(
             "            \"ExposedPorts\": {\n"
-                + "                \"1000\": {},\n"
+                + "                \"1000/tcp\": {},\n"
```

Comment:
should we keep one in that doesn't have a protocol associated with it?

Reply:
We still allow empty protocol configuration in jib (which defaults to TCP), but I felt it was best to keep it explicit in the backend so we don't have to depend on other peoples' defaults.

Reply:
Oh, I see, we're just writing in tcp if the user input is just a port. :+1: